content/en/docs/security/_index.md

```diff
@@ -1,4 +1,19 @@
 ---
 title: Security
-weight: 150
+description: How to use OpenTelemetry Collector securely.
+weight: 10.1
 ---
+
+When setting up the OpenTelemetry (OTel) Collector, consider implementing
+security best practices in both your hosting infrastructure and your OTel
+Collector configuration.
+
```

Comment:
Could we explain briefly in the intro what's at stake here? Why should someone care about securing the OpenTelemetry Collector? Is it a painful process? What knowledge is required?

Reply:
You ask great questions @theletterf! If you have the answers, I'm happy to put them in this document. I defer to the SMEs to answer these questions otherwise. At the very least, I will file issues if these questions aren't answered or incorporated by the time we merge this PR.